environments/illumina/environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
   - bioconda
   - defaults
+  - r
```

Comment:
r or r-devel? the latest version or specific R version?

Reply:
I have no strong opinion, I am not sure what is best, all I can say is if we have it as we have right now it works on our cluster. If you think r-devel is better, then we could have it, but we should make sure it works first.

Reply:
anyhow, I think it is a good idea to specify version, will make things run faster and we might avoid annoying surprises in the future if something change, any suggestions what version?

Reply:
Shall I build the container then with 4.1.3 and upload to GMS dockerhub?

Reply:
@sofstam I think it would be nice if you please add that. :)

Reply:
I am getting the following error when trying to build the docker and add r=4.1

```
#13 109.5 Encountered problems while solving:
#13 109.5   - package biopython-1.74-py27h7b6447c_0 requires python >=2.7,<2.8.0a0, but none of the providers can be installed
```

Do you get any similar error @JD2112 ?

Reply:
Hi @sofstam Sorry for late reply, here is the link for R-4.1.3 from Honda [https://anaconda.org/conda-forge/r-base](https://anaconda.org/conda-forge/r-base) . Please let me know if it works.

Reply:
I am getting the following error when trying to build the docker and add r=4.1

#13 109.5 Encountered problems while solving:
#13 109.5   - package biopython-1.74-py27h7b6447c_0 requires python >=2.7,<2.8.0a0, but none of the providers can be installed
Do you get any similar error @JD2112 ?

About the biopython error, I think it is because of conflicts between python versions. Might works if biopython is installed using the pip. What do you think?